README.md

```diff
@@ -135,6 +135,17 @@ sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more infor
 
 Similar to configuring the server, the client configuration file is located at `/etc/opt/omi/conf/omicli.conf`.
 
+### Collecting Performance Metrics using OMI/OMI Provider on docker container or virtual machine
+
+OMI/OMI provider can work on containers and machines, just make sure you know following notes:
+- OMI/OMI provider can collect virtual machine perf metrics even in the normal container. To do this, there are 3 ways docker provided: pseudo-files, docker stats command and Remote API, and it doesn't need to run with a privileged container, . 
+- Multiple OMI/OMI providers can run in different containers or/and inside VM if they are mapped into different ports of the host. These instances can simultaneously collect VM-based perf metrics without any issue.
+- By default OMI/OMI provider cannot collect container perf metrics. 
+  - To be able to do that, you need to implement a provider something similar as [ContainerStatisticsProvider] or you can just use the implemented provider.
```

Comment:
Change this to:

- To be able to collect container performance metrics, you can use [ContainerStatisticsProvider], or you can implement a similar provider. If you wish to use [ContainerStatisticsProvider], this ships with [OMS]. You can either install [OMS] itself, or you can extract [ContainerStatisticsProvider] from the [OMS] bundle and install that provider independently.